### DISCUSSION TITLE
How to set the gastric emptying time in Populations in PK-sim v11.2?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1567] [Questions & Problems]
I have assumed gastric emptying to be 45min in Individuals and then set Populations based on Individuals, resulting in a large difference in PK between the two scenario modes. The gastric emptying time is shown in the figure, and I want to modify the gastric emptying in Populations to be concentrated at 45min or normally distributed centered at 45min. If I can't modify it, then how can I get the 95% confidence interval for the results simulated with Individuals?

![1](https://github.com/Open-Systems-Pharmacology/Forum/assets/109507330/f71009ec-68c4-4592-ba60-375b5ea9f726)
![2](https://github.com/Open-Systems-Pharmacology/Forum/assets/109507330/c76f7749-5c40-4282-bd34-c2a56b7dd925)

**Comment by Yuri05** [comment]
The gastric emptying time has a **lognormal** distribution by default. 
You cannot change this directly in PK-Sim. If you want a different type of distribution, you can export your population to a CSV file, adjust the GET values to suit your needs and then re-import the population from the CSV file.
